server/zanata-war/src/test/resources/org/zanata/adapter/test-gettext-comments.pot

```diff
@@ -0,0 +1,16 @@
+#  translator-comments
+#. comments-given-by-programmer
+#: reference
+#, flag
+msgid ""
```

Comment:
Make the first entry Line One.
You should have a differing comment on the first and second entries and test them both.